NOTES.create-cmd_states.db3

```diff
@@ -1,4 +1,9 @@
-To make the fresh sqlite3 version of relevant command states Sybase tables:
+To make a fresh sqlite3 version of command states Sybase tables:
 
-$ ./make_new_tables.py --tstart=1999:001 --cmds --cmd-states  # takes a few minutes
+$ ./make_new_tables.py --tstart=1999:001 --cmds --cmd-states # takes a few minutes
+
+To make a complete sqlite3 version to use for on-the-side sqlite3 crontasks, do:
+
+# ./make_new_tables.py --tstart=1999:001 --cmds --cmd-states --tl_processing # takes a few minutes
```

Comment:
Why are there two versions of making a baseline new table?  I don't know quite what an "on-the-side sqlite3 crontask" is.  And why is this second command commented out?